doglist.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/style.css">
+    <title>All the Dogs</title>
+  </head>
+  <body>
+    <header class="head">
+      <div class="checkout">
+        <a href="checkout.html">
+          <img src="images/cart-icon.svg" alt="Cart Icon" >
+          $1234.56
+        </a>
+        <input type="button" name="cart" value="checkout">
+      </div>
+      <img src="images/adoptadog-logo.svg" alt="Adoptadog logo">
+      <p> Seattle’s premiere dog adoption agency </p>
+
+    </header>
+
+    <nav class="top">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="doglist.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+
+    <div class="top-text2">
+        <h2>Here are all of our Dogs!</h2>
+    </div>
+
+    <div class="container1">
```

Comment:
I like how you're using wrapping containers all over your code. I'd recommend using more semantic class names. It'll make it easier to remember what things do. instead of container1 maybe something like dogs-container, etc.